resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -363,6 +373,28 @@ public Builder slowCallDurationThreshold(Duration slowCallDurationThreshold) {
             return this;
         }
 
+        /**
+         * Configures CircuitBreaker with a fixed wait duration which controls how long the
+         * CircuitBreaker should stay in Half Open state, before it switches to open. This is an
+         * Optional parameter.
+         *
+         * By default circuitBreaker will stay in Half Open state until
+         * {@code minimumNumberOfCalls} is completed with either success or failure.
+         *
+         * @param waitDurationInHalfOpenState the wait duration which specifies how long the
+         *                                CircuitBreaker should stay in Half Open
+         * @return the CircuitBreakerConfig.Builder
+         * @throws IllegalArgumentException if {@code waitDurationInOpenState.toMillis() < 1000}
+         */
+        public Builder waitDurationInHalfOpenState(Duration waitDurationInHalfOpenState) {
+            if (waitDurationInHalfOpenState.toMillis() < 1000) {
```

Comment:
I think `waitDurationInHalfOpenState` can be smaller than 1000ms like `waitDurationInOpenState`.
https://github.com/resilience4j/resilience4j/blob/13bdd4b51e1b216670afcab7cf2d3861213971c7/resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java#L322-L330